_data/design/government-of-canada-design-system.yml

```diff
@@ -48,6 +48,6 @@ administrations:
         team:
           en: Digital Transformation Office
           fr: Bureau de la transformation numérique
-designStatus: 
-  en: Original design
-  fr: Design original
+        designStatus: 
```

Comment:
Is this supposed to be nested under the administration?

Reply:
Yes, I made a mistake when originally setting it up, and it seemed easier to just tab it directly than resubmit the entire form